jts-test-runner/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "jts-test-runner"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+approx = "0.4.0"
+geo = { path = "../geo" }
+include_dir = { version = "0.6", features = ["search"] }
+log = "0.4.14"
+serde =  { version = "1.0.105", features = ["derive"] }
+serde-xml-rs = "*"
```

```suggestion
serde-xml-rs = "0.5"
```

(or `0.5.0` or `0.5.1`, if you prefer)

---

But we can change this later on, no need to do it in this PR.

---

Good catch! updated.